docs/Agent/installation/ansible_based_installation.md

```diff
@@ -0,0 +1,37 @@
+# Ansible based installation
+
+In the following, we describe how to install Pbench Agent using an ansible playbook.
+
+:::{note}
+The same Pbench Agent version must be installed on all the test systems that participate in a benchmark run, there is no support for mixed installations.
```

Comment:
That comma should be a colon or m-dash (or period).

Reply:
I wasn't sufficiently specific...my apologies, @vishalvvr!

The comma at line 3 was fine; it's line 6 which needs the change.  (Please change line 3 back.... 😊)

Reply:
Line 3:  remove the colon, and possibly remove the "In the following" phrase, since it's kind of obvious.  If the result doesn't read well, try something like "This document describes" (which allows us to escape using the first-person pronoun).

Line 6:  the comma should be replaced by a colon or m-dash (or period).